packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
```

Comment:
ah right. if you're still around this code, can you add a comment to that effect? no big deal if you're not there any more.